### PR TITLE
ci: updated k8s versions

### PR DESCRIPTION
**Pull request by kvanzuijlen**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -58,10 +58,11 @@ jobs:
         k8s:
           # from https://github.com/yannh/kubernetes-json-schema
           - v1.24.17
```
Comment (viceice) [inline .github/workflows/ci.yaml:60]: I think we should drop v1.24

**Review by viceice — COMMENTED** [review]
I think we should drop v1.24 and v1.25

**Comment by kvanzuijlen** [comment]
> I think we should drop v1.24 and v1.25

1.24 is going EoL today on GKE (Google Cloud) and end of January on EKS (Amazon AWS). 1.25 will go EoL on GKE end of February and May on EKS. If I should still drop both, please let me know!

**Comment by viceice** [comment]
so it seems safe to me to drop those now as almost all users should already be on higher versions.

**Review by viceice — APPROVED** [review]

**Comment by viceice** [comment]
what's your opinion on #597?

**Pull request merged**